src/components/libraryHeader1.tsx

```diff
@@ -0,0 +1,118 @@
+import styled from '@emotion/styled';
+import { Link } from 'react-router-dom';
+
+export default function LibraryHeader1() {
+  return (
+    <Header>
+      <LeftContainer>
+        <Link to={'/'} style={{ textDecoration: 'none' }}>
+          <LogoContainer>
+            <LogobarIcon src="long Black bar.svg" alt="logo_blackbar"></LogobarIcon>
+            <LogoTitle>brunch story</LogoTitle>
+          </LogoContainer>
+        </Link>
+
+        <PageLinker>최근 업로드 글</PageLinker>
+        <ClickedPageLinker>최근 읽었던 글</ClickedPageLinker>
+        <PageLinker>내가 구독한 글</PageLinker>
+      </LeftContainer>
+
+      <RightContainer>
+        <WriteButton>
+          <WriteContent>글쓰기</WriteContent>
+        </WriteButton>
+        <ProfileImg />
+        <SearchIcon src="icn_search.png" alt="searchIcon"></SearchIcon>
+      </RightContainer>
+    </Header>
+  );
+}
+
+const Header = styled.header`
+  position: absolute;
+  left: 0rem;
+  top: 0rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 15px 14px;
+  width: 100%;
+`;
+
+const LeftContainer = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  align-items: flex-end;
+  gap: 8px;
+`;
+
+const LogoContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  color: ${({ theme }) => theme.color.gray11};
+`;
+const LogobarIcon = styled.img`
+  position: relative;
+  left: 48px;
+  top: 6px;
+  width: 50%;
+  height: 3px;
+`;
+
+const LogoTitle = styled.p`
+  font: ${({ theme }) => theme.font.body2};
+`;
```

Comment:
p3: 폰트는 font 앞에 안써도 가져올 수 있게 해놨어!
```suggestion
const LogoTitle = styled.p`
  ${({ theme }) => theme.font.body2};
`;
```

Reply:
헉 오키오키 땡큐